python/lsst/pipe/tasks/processCcd.py

```diff
@@ -124,13 +124,15 @@ class ProcessCcdTask(pipeBase.CmdLineTask):
     Add the option `--help` to see more options.
     """
     ConfigClass = ProcessCcdConfig
+    RunnerClass = pipeBase.ButlerInitializedTaskRunner
     _DefaultName = "processCcd"
 
     def __init__(self, **kwargs):
+        butler = kwargs.pop('butler')
         pipeBase.CmdLineTask.__init__(self, **kwargs)
         self.makeSubtask("isr")
-        self.makeSubtask("charImage")
-        self.makeSubtask("calibrate", icSourceSchema=self.charImage.schema)
+        self.makeSubtask("charImage", butler=butler)
+        self.makeSubtask("calibrate", icSourceSchema=self.charImage.schema, butler=butler)
```

Comment:
If you do decide to list butler before icSourceSchema in the arg list of the calibrate task, I suggest using the same order in calling, even though it doesn't really matter. It just makes the code a bit easier to read.